standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -865,8 +911,10 @@ Partition appendPartition(String dbName, String tableName, List<String> partVals
    * @throws MetaException error accessing the RDBMS
    * @throws TException thrift transport error
    */
-  Partition appendPartition(String catName, String dbName, String tableName, List<String> partVals)
-      throws InvalidObjectException, AlreadyExistsException, MetaException, TException;
+  default Partition appendPartition(String catName, String dbName, String tableName, List<String> partVals)
```

Comment:
should throw UnsupportedOperationException

Reply:
Done.